src/main/java/org/broadinstitute/hellbender/tools/funcotator/dataSources/DataSourceUtils.java

```diff
@@ -63,19 +65,18 @@ private DataSourceUtils() {}
     @VisibleForTesting
     static final int MIN_YEAR_RELEASED        = 2018;
     @VisibleForTesting
-    static final int MIN_MONTH_RELEASED       = 6;
+    static final int MIN_MONTH_RELEASED       = 8;
     @VisibleForTesting
-    static final int MIN_DAY_RELEASED         = 15;
+    static final int MIN_DAY_RELEASED         = 29;
 
     //==================================================================================================================
     // Public Static Members:
 
     /** The minimum version of the data sources required for funcotator to run.  */
     public static final String CURRENT_MINIMUM_DATA_SOURCE_VERSION         = String.format("v%d.%d.%d%02d%02d", MIN_MAJOR_VERSION_NUMBER, MIN_MINOR_VERSION_NUMBER, MIN_YEAR_RELEASED, MIN_MONTH_RELEASED, MIN_DAY_RELEASED);
-    public static final String README_FILE_NAME                            = "README.txt";
+    public static final String MANIFEST_FILE_NAME                          = "MANIFEST.txt";
```

Comment:
Did you just rename README.txt to MANIFEST.txt, or is this a more substantial change? Ie., is `MANIFEST.txt` now a structured format, or is it still basically a README with another name?

Reply:
@droazen I added a `MANIFEST.txt` file to the packages.  `MANIFEST.txt` contains the version and provenance information for each data source package, as well as a new decorator/information field to distinguish somatic and germline files.  This version information is duplicated in the readme, but the readme is not parsed.

`MANIFEST.txt` a plain-text formatted file with 1 field per line.  The formatting is loosely enforced with the regex in `DataSourceUtils` and is of the syntax:

<FIELD>: <VALUE>

Reply:
Ok, sounds good @jonn-smith, thanks for the clarification.